src/components/ActNowPage/index.js

```diff
@@ -111,7 +114,7 @@ function ActNow({ actNow, footer, title, navigation, description, ...props }) {
       >
         <div className={classes.contentSection}>
           <ContentSection
-            contentProps={{ lg: 9, title: null }}
+            contentProps={{ lg: 7, title: null }}
```

Comment:
Content has 9 columns (1 is empty so 8 should work. 7 is too small).
![Screenshot 2021-07-23 at 09-04-43 PT_desktop](https://user-images.githubusercontent.com/1779590/126743254-cc57a884-0a04-48de-bc16-9652d777a8a9.png)

Reply:
Sure. What are you using to see the highlights above?

Reply:
Layout Grid option in the XD 
![Peek 2021-07-23 09-45](https://user-images.githubusercontent.com/1779590/126746448-136fde7d-c8b3-40f6-807f-35c117caf8aa.gif)